cmd/build.go

```diff
@@ -64,7 +68,6 @@ func newLocalExecuteCommand(config *settings.Config) *cobra.Command {
 		RunE: func(_ *cobra.Command, _ []string) error {
 			return runExecute(opts)
 		},
-		DisableFlagParsing: true,
```

Comment:
Disabling flags is necessary here. There's a short comment about this above the flags, but I guess we should add one above this line as well. The flags need to be handled by the process in the container, so we need them to be available from `args`. When flags are parsed by a command they are not provided from `args []string`.

We should add an inline comment about why this is disabled. We want these flags to exist on the command so that the `help` command works properly.

Reply:
Did you run my PR? This was done because without it, any flags that the user would want to pass to Docker won't work because Cobra would try to interpret it as a Cobra flag for the CircleCI CLI. It's my understanding this is why this line (67) was added.

The Cobra flags can be separated from the Docker flags by a double hyphen. Anything before the double hyphen gets interpreted by Cobra as normal, and anything after gets passed to Docker, which was the reason for this line in the first place.